test/test_uri_parser.py

```diff
@@ -294,32 +293,6 @@ def test_parse_uri(self):
         self.assertEqual(res, parse_uri("mongodb://localhost/?readPreference=secondary"))
 
         # Various authentication tests
-        res = copy.deepcopy(orig)
-        res["options"] = {"authmechanism": "MONGODB-CR"}
```

Comment:
Rather than removing these tests, can you update then to use SCRAM-SHA-256?

Reply:
Done